.env.appStore.example

```diff
@@ -125,11 +125,4 @@ SALESFORCE_CONSUMER_SECRET=""
 ZOHOCRM_CLIENT_ID=""
 ZOHOCRM_CLIENT_SECRET=""
 
-# ALBY
-# Used for the Alby payment app / receiving Alby payments
-# Get it from: https://getalby.com/developer/oauth_clients
-# Set callbackUrl to YOUR_APP_DOMAIN/api/integrations/alby/alby-webhook
-NEXT_PUBLIC_ALBY_CLIENT_ID=""
```

Comment:
These were removed but not from turbo.json and `packages/app-store/alby/pages/setup/index.tsx` needs to be updated to use the client_id and client_secret properties

Reply:
fixed